packages/create-gatsby/src/tracking.ts

```diff
@@ -0,0 +1,31 @@
+import fetch from "node-fetch"
+
+export interface ITrackCliArgs {
+  name?: string
+  valueString?: string
+  exitCode?: number
+  valueStringArray?: Array<string>
+}
+
+const analyticsApi =
+  process.env.GATSBY_TELEMETRY_API || `https://analytics.gatsbyjs.com/events`
+
+const getUserAgent = (): string => {
+  try {
+    const { name, version } = require(`../package.json`)
+    return `${name}:${version}`
+  } catch (e) {
+    return `Gatsby Telemetry`
+  }
+}
+
+export const trackCli = (eventName: string, args?: ITrackCliArgs): void => {
```

Comment:
ℹ️ this is the lightweight version of track cli for now.

I would need some help to get the shape of the event to send there 😞 cc @ascorbic @jamo